.github/actions/spell-check/dictionary/dictionary.txt

```diff
@@ -143898,6 +143898,8 @@ first-seen
 firstship
 first-string
 first-told
+firstuse
```

Comment:
Wait this one actually does seem like a typo, we probably just need to fix this one

Reply:
This one is in the URL https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse . I don't even understand why it will check things inside an URL...

Reply:
Oh weird, I thought the spellcheck bot ignored URLs...

Reply:
Ah I see, it only ignores _youtube_ URLs. That should probably be updated for at the very least `docs.microsoft.com/.*`, urls, if not all of them

Reply:
I need to post one more PR to update the action so that on passing runs there's feedback on unnecessary whitelist items. It'll probably be my last PR for over a week. (Holy days)